app/scripts/util/createDispatcher.js

```diff
@@ -0,0 +1,64 @@
+/**
+ *  RxJS in action
+ *  Chapter #
+ *  @author Paul Daniels
+ *  @author Luis Atencio
+ */
+
+(function(root) {
+
+// A custom utility operator for only accepting messages
+// of a certain type
+  function ofType(target) {
+    return source => source.filter(({type}) => type === target);
+  }
+
+  function createDispatcher(store) {
```

Comment:
This is definitely going to take some explaining from a "high level" conceptual area, however, from the low level these should all be concepts that they have seen.